plugin/src/sbt-test/scalafmt-sbt/sbt/project/build.properties

```diff
@@ -0,0 +1 @@
+sbt.version=1.2.1
```

Comment:
Need to delete this as it'll filter the test out if not

Reply:
Weird, not sure where this file came from. Will delete.

Reply:
I know exactly because I implemented.

As of sbt... 1.1? maybe 1.2, if sbt doesn't find an sbt.version it'll will write out its running version into project/build.properties, which is a best practice.  However, someone else implemented that that version in a scripted test acts as a filter.  So you ran sbt and it wrote the file.  It's bad, so I got round to filing the issue: https://github.com/sbt/sbt/issues/4536

Reply:
The auto-created build.properties is often nice but I've found it sometimes annoying when I start `sbt` in the wrong directory. In Metals we use the following heuristics to determine if a directory is an sbt build

- `build.sbt` exists, or
- `project/build.properties` exists and defines `sbt.version` property

I've hit on cases where auto-generated build.properties file cause this heuristic to fail.

Reply:
@cb372 Let's remove this file

Reply:
@olafurpg I agree, and it's one of the reasons I prefer sbt-extras, which has a similar heuristic to that.